p2p/transport/webrtc/udpmux/muxed_connection.go

```diff
@@ -0,0 +1,164 @@
+package udpmux
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net"
+	"time"
+
+	pool "github.com/libp2p/go-buffer-pool"
+)
+
+var _ net.PacketConn = &muxedConnection{}
+
+const maxPacketsInQueue int = 128
```

Comment:
No need to make this a typed constant.
```suggestion
const maxPacketsInQueue = 128
```